check-signatures-format.py

```diff
@@ -7,12 +7,12 @@
 ok = True
 for file_name in sorted(os.listdir("_data/signed")):
     if not file_name.endswith(".yaml") or file_name[:-5] != file_name[:-5].strip():
-        print(file_name, "has invalid name")
+        print(file_name, "probably has an invalid name. Please wait for review or use a real name instead.")
         ok = False
     with open(f"_data/signed/{file_name}") as f:
         contents = f.read()
     if not re.fullmatch(regex, contents):
-        print(file_name, "has invalid format")
+        print(file_name, "has invalid format. Please check the format example on README")
```

Comment:
Print out correct format here with a suggestion to change file in their repo.

Reply:
OK done.

Reply:
Perhaps we need a bit more parsing than just a regex check. I'd recommend something along the lines of:

- Ask to remove all empty lines, possibly except the last. Also there are ~10% PRs that add *two* empty lines to the end, let's allow that too.
- Check that each line contains a :, ask the user to add `name:` or `link:` before it otherwise.
- Check that the only keys are `name:` and `link:`, not e.g. `mail:`.
- Check that the order is `name` `link`, not `link` `name`.
- Check that there are no two spaces in a row, no line ends with a space and there's a space after each colon.
- Check the link. If it looks like a email but doesn't start with `mailto:`, ask the user to add it.